docs/platforms/esp32/vs_code_development.md

```diff
@@ -18,7 +18,9 @@ Follow
 [these](https://github.com/project-chip/connectedhomeip/edit/master/docs/VSCODE_DEVELOPMENT.md)
 steps to set up Visual Studio Code for Matter development.
 
-### Linux <a name="linux">
+(linux)=
```

Comment:
this seems off ... what is `(linux)=`? i tried to view the "rich diff" and it does indeed that this shows up verbatim and if so, it looks odd.

Reply:
So the difference would be github markdown vs our generated doc? 

I guess ok, however ideally we should minimize the use of this to not confuse github too much...